src/components/AudioPlayer/Player.js

```diff
@@ -419,6 +419,13 @@ class AudioPlayer extends Component {
       right: timePosition === 'right' ? rightIconsWidth + 10 : 'auto'
     }
 
+    let timeRanges = []
+    if (buffered && !!buffered.length) {
```

Comment:
unnecessary boolean forcing (`!!`) only needed in a react context because `false` mean no child and e.g. `0` (falsy) is rendered as text `0`